src/main/java/seedu/address/model/person/passenger/Passenger.java

```diff
@@ -75,10 +75,18 @@ public TripDay getTripDay() {
         return tripDay;
     }
 
+    public String getTripDayStr() {
+        return tripDay.toString();
+    }
+
     public TripTime getTripTime() {
         return tripTime;
     }
 
+    public String getTripTimeStr() {
```

Comment:
small thing but, perhaps this could be renamed getTripTimeAsString() as we have discussed